src/main/java/com/synopsys/integration/detect/configuration/DetectProperties.java

```diff
@@ -455,6 +457,19 @@ private DetectProperties() {
                 "Use this value to enable the various snippet scanning modes. For a full explanation, please refer to the 'Running a component scan using the Signature Scanner command line' section in your Black Duck server's online help. Corresponding Signature Scanner CLI Arguments: --snippet-matching, --snippet-matching-only, --full-snippet-scan.")
             .setGroups(DetectGroup.SIGNATURE_SCANNER, DetectGroup.GLOBAL, DetectGroup.SOURCE_SCAN)
             .build();
+    
+    public static final ExtendedEnumProperty<ExtendedReducedPersistanceMode, ReducedPersistence> DETECT_BLACKDUCK_SIGNATURE_SCANNER_REDUCED_PERSISTENCE =
+            ExtendedEnumProperty.newBuilder(
+                    "detect.blackduck.signature.scanner.reduced.persistence",
+                    ExtendedEnumValue.ofExtendedValue(ExtendedReducedPersistanceMode.DEFAULT),
+                    ExtendedReducedPersistanceMode.class,
+                    ReducedPersistence.class
+                )
+                .setInfo("Reduced Persistence", DetectPropertyFromVersion.VERSION_8_3_0)
+                .setHelp(
+                    "Use this value to control how unmatched files from signature scans are stored. For a full explanation, please refer to <xref href=\"https://community.synopsys.com/s/document-item?bundleId=bd-hub&topicId=ComponentDiscovery%2Fabout_reduced_persistence_signature_scanning.html&_LANG=enus\" scope=\"external\" outputclass=\"external\" format=\"html\" target=\"_blank\">about reduced persistence signature scanning.</xref>")
+                .setGroups(DetectGroup.SIGNATURE_SCANNER, DetectGroup.GLOBAL)
```

Comment:
@sig-cpotts How stable will a URL like this be? (If not very) is there a more reliable way to point users to Black Duck documentation pages?

Reply:
Hey @stevebillings - David and I talked about this quandary. With docs moving to a new Zoomin Documentation Portal this is at least temporarily the most sane. I have created a task for myself to revisit once the doc migration completes.